src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -231,6 +234,12 @@ private HeldPiece getColorSensorPiece() {
         gEntry.setValue(green);
         bEntry.setValue(blue);
 
+        //if the color is 0 0 0, the sensor is broken and should be rebooted
+        //only do this every couple seconds
+        if(colorTimer.advanceIfElapsed(2) && red == 0 && green == 0 && blue == 0){
```

Comment:
this timer doesn't seem to be started. also we should immediately reboot if 0,0,0 but not try again until after 2 seconds if it's still broken